src/modules/data/useData.ts

```diff
@@ -0,0 +1,18 @@
+import { useFirestore, useFirestoreCollectionData } from 'reactfire';
+
+const useData = <T, D>(COLLECTION_NAME: string) => {
```

Comment:
How about `useCollectionData`

```suggestion
const useCollectionData = <T, D>(COLLECTION_NAME: string) => {
```